html/admin/index.html

```diff
@@ -110,13 +113,27 @@ <h1>Admin Password</h1>
         </form>
     </div>
 
+    <div class="container" id="banner" style="display: none;">
+      <div class="page-header">
+        <h1>User Interface</h1>
+      </div>
+      <p class="lead">Banner Message</p>
```

Comment:
As this takes HTML, let’s make that clear by changing this to “Banner Message (html or plain text)"